tests/page-objects/default/sms/messages.wdio.page.js

```diff
@@ -50,6 +50,7 @@ const getMessageContent = async  (index = 1) => {
   return {
     content: await sms.$('p[test-id="sms-content"]').getText(),
     state: await sms.$('.state').getText(),
+    dataID: await sms.getAttribute('data-id'),
```

Comment:
```suggestion
    dataId: await sms.getAttribute('data-id'),
```

Super minor, but usually I see it like this when camel-casing "ID".